src/context/notesContext.js

```diff
@@ -0,0 +1,9 @@
+import { useContext, createContext, useState } from "react";
+const NotesContext = createContext(null);
+const useNotes = () => useContext(NotesContext);
+
+const NotesProvider = ({children}) => {
+  const [Allnotes, setAllnotes] = useState([]);
```

Comment:
Use proper varible naming conventions. like use camelCase throughout the project